src/adapter/threads.ts

```diff
@@ -868,37 +869,52 @@ export class Thread implements VariableStoreDelegate {
       (source as any)[kScriptsSymbol] = new Set();
     (source as any)[kScriptsSymbol].add(script);
 
-    if (!this._pauseOnSourceMapBreakpointId && event.sourceMapURL) {
```

Comment:
This is a somewhat unrelated thing I ran into when testing with an Electron apps. Electron apps are generally an 'attach' scenario, and we don't seem to hit `beforeScriptWithSourceMapExecution` when the scripts are already being executed, even if we're able to set the breakpoint. I adjusted this so that we kick off the source map loading (and eventually `_scriptWithSourceMapHandler`) when we get the loaded event, and just await that promise if/when we hit an instrumentation breakpoint for the script.